<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alarm_plugin::AlarmsListDialog</class>
 <widget class="QDialog" name="alarm_plugin::AlarmsListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alarm Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="7">
    <widget class="QListWidget" name="alarms_list">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="add_btn">
     <property name="toolTip">
      <string>Add new alarm</string>
     </property>
     <property name="text">
      <string notr="true">...</string>
     </property>
     <property name="icon">
      <iconset resource="../alarm.qrc">
       <normaloff>:/alarm/add.svg.p</normaloff>:/alarm/add.svg.p</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="del_btn">
     <property name="toolTip">
      <string>Delete selected</string>
     </property>
     <property name="text">
      <string notr="true">...</string>
     </property>
     <property name="icon">
      <iconset resource="../alarm.qrc">
       <normaloff>:/alarm/del.svg.p</normaloff>:/alarm/del.svg.p</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>70</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="disable_all_btn">
     <property name="toolTip">
      <string>Disable all alarms</string>
     </property>
     <property name="text">
      <string notr="true">...</string>
     </property>
     <property name="icon">
      <iconset resource="../alarm.qrc">
       <normaloff>:/alarm/disable_all.svg.p</normaloff>:/alarm/disable_all.svg.p</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QToolButton" name="delete_all_btn">
     <property name="toolTip">
      <string>Remove all alarms</string>
     </property>
     <property name="text">
      <string notr="true">...</string>
     </property>
     <property name="icon">
      <iconset resource="../alarm.qrc">
       <normaloff>:/alarm/remove_all.svg.p</normaloff>:/alarm/remove_all.svg.p</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>70</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <widget class="QToolButton" name="adv_settings_btn">
     <property name="toolTip">
      <string>Advanced settings</string>
     </property>
     <property name="text">
      <string notr="true">...</string>
     </property>
     <property name="icon">
      <iconset resource="../alarm.qrc">
       <normaloff>:/alarm/settings.svg.p</normaloff>:/alarm/settings.svg.p</iconset>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../alarm.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>alarm_plugin::AlarmsListDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>alarm_plugin::AlarmsListDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
